frontend/src/components/posts/Feed.js

```diff
@@ -3,46 +3,140 @@ import { connect } from "react-redux";
 import { Link } from "react-router-dom";
 import PropTypes from "prop-types";
 import { getPosts, deletePost } from "../../actions/posts";
+import { checkFollower, addFollower} from "../../actions/followers";
+
 import Moment from "react-moment";
 import { FaRegClock, FaTrashAlt } from "react-icons/fa";
 
+import Dialog from '@mui/material/Dialog';
```

Comment:
you cant use a bootstrap modal? this is a lot of dependencies for one dialog

Reply:
i tried using that, i will need to download a package for that too, and it was giving me errors.